api/runtime_env.proto

```diff
@@ -58,13 +58,13 @@ message DescribeRuntimeEnvsRequest{
 	google.protobuf.StringValue search_word = 3;
 	google.protobuf.StringValue selector = 4;
 	repeated string owner = 5;
-	google.protobuf.UInt32Value limit = 6;
-	google.protobuf.UInt32Value offset = 7;
-	google.protobuf.UInt32Value verbose = 8;
+	uint32 limit = 6;
+	uint32 offset = 7;
+	uint32 verbose = 8;
```

Comment:
There is no comparison to use `verbose` here.

Reply:
One question is whether to return runtime_env_credential info when I query runtime_env.Or do I just return all the info?

Reply:
We return runtime_env_credential by default, and the first phase does not seem necessary to add `verbose`.